dev_guide/cron_jobs.adoc

```diff
@@ -127,14 +129,30 @@ Doing this prevents them from generating unnecessary artifacts.
 [[cleaning-up-after-a-cron-job]]
 == Cleaning Up After a Cron Job
 
-Cron jobs can leave behind artifact resources such as jobs or pods. Check if any remain:
+Cron jobs can leave behind artifact resources such as jobs or pods.  As a user it is important
+to configure history limits so that old jobs and their pods are properly cleaned.  There are
+two optional fields within cron job's spec responsible for that:
 
+[source,yaml]
 ----
-$ oc get jobs
-$ oc get pods
+apiVersion: batch/v2alpha1
+kind: CronJob
+metadata:
+  name: pi
+spec:
+  successfulJobsHistoryLimit: 3 <1>
+  failedJobsHistoryLimit: 1     <2>
+  schedule: "*/1 * * * *"
+  jobTemplate:
+    spec:
+  ...
 ----
 
-All artifacts left over from a job execution use the job name as their prefix. For example, given the cron job example:
+1. The number of successful finished jobs to retain.
+2. The number of failed finished jobs to retain.
```

Comment:
I think the above two numbers needs to be written up differently, so <1> and <2>. Not sure if it affects the readout, though.